src/ResourceManagement/HDInsight/HDInsight.Tests/ScenarioTests/CreateClusterTests.cs

```diff
@@ -235,6 +235,7 @@ public void TestCreateLinuxClusterWithPremiumTier()
                 var resourceGroup = HDInsightManagementTestUtilities.CreateResourceGroup(resourceManagementClient);
 
                 var cluster = GetClusterSpecHelpers.GetCustomCreateParametersIaas();
+                cluster.Version = "3.4";
```

Comment:
this is the version of the HDInsight cluster. This is the same as the HDI release version that we announce publicly.